packages/preview-middleware/README.md

```diff
@@ -185,3 +185,36 @@ return flp.router
 - `rootProject` - [Reader](https://sap.github.io/ui5-tooling/stable/api/@ui5_fs_AbstractReader.html) to access resources of the root project
 - `middlewareUtil` - [MiddlewareUtil](https://sap.github.io/ui5-tooling/v3/api/@ui5_server_middleware_MiddlewareUtil.html) of the UI5 server
 - `logger` - Logger instance for use in the middleware.
+
+
+## [Migration](#migration)
+In case you have no custom modifications in the local fiori launchpad sandbox files (`webapp/test/flpSandbox.html` or `webapp/test/flpSandboxMockserver.html`) you're already good to go.
+
+If you have custom modifications in the local fiori launchpad sandbox files,
```

Comment:
```suggestion
If you have custom modifications in the local Fiori Launchpad sandbox files, you need to migrate them into a custom `.js` or `.ts` file (depending on your setup) and integrate this file as a custom `init` script into the configuration options of the middleware.
```

Reply:
Done in fe3e4ffa7a57333062a7629ce48b4f040708751f